corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -38,30 +39,71 @@
     get_single_worksheet,
 )
 
+from corehq.util.files import read_workbook_content_as_file
 
-def validate_bulk_app_translation_upload(app, workbook, email, lang_to_compare):
+from CommcareTranslationChecker import validate_workbook
+from CommcareTranslationChecker.exceptions import FatalError
+
+
+def run_translation_checker(file_obj):
```

Comment:
nit: I was suggested to use the Step Down rule which I find a good practice to follow, https://dzone.com/articles/the-stepdown-rule